frontend/src/Components/SignupBox.jsx

```diff
@@ -0,0 +1,212 @@
+import PhoneInput, {isValidPhoneNumber} from 'react-phone-number-input'
+import countryNames from 'react-phone-number-input/locale/en'
+import {jwtDecode} from 'jwt-decode';
+
+const SignupBox = () =>{
+    const [username, setUsername] = useState('');
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [confermPassword, setConfermPassword] = useState("");
+    const [country, setCountry] = useState("+20");
+    const [phone, setPhone] = useState("");
+    const [city, setCity] = useState("");
+    const [address, setAddress] = useState("");
+    const [gender, setGender] = useState("");
+    const [errorMessage, setErrorMessage] = useState('');
+    const [errorTrigger, setErrorTrigger] = useState('');
+    const navigate = useNavigate();
+
+    const handleCallbackResponse = async (response)=>{
+        var user = jwtDecode(response.credential);
+        const checkUserEmail = await fetch(`http://localhost:8081/users?email=${user.email}`)
+        .then(response=>response.json())
+        .then(async (data)=>{
+            console.log(data, user.email);
+            if (data.length == 0){
+                const checkAdminEmail = await fetch(`http://localhost:8081/admins?email=${user.email}`)
+                .then(response=>response.json())
+                .then(async (data)=>{
+                    if (data.length == 0){
+                        console.log(user);
+                        const register = await fetch(`http://localhost:8081/users/`,{
+                            method: 'POST',
+                            headers:{
+                                'Content-Type': 'application/json'
+                            },
+                            body: JSON.stringify({
+                                username: user.name,
+                                email: user.email,
+                                password: '',
+                                phone: '',
+                                country: '',
+                                city: '',
+                                address: '',
+                                gender: '',
+                                picture: user.picture
+                            })
+                        })
+                        .then(Response=>Response.status==201 ? navigate('/homepage', 
+                            {state: {user: {
+                                                username: user.name,
+                                                email: user.email,
+                                                password: '',
+                                                phone: '',
+                                                country: '',
+                                                city: '',
+                                                address: '',
+                                                gender: '',
+                                                picture: user.picture
```

Comment:
make picture field empty

Reply:
Ok, but you will have to save the photo's path in the local folder in the server